xml/cap_admin_app_domains.xml

```diff
@@ -0,0 +1,145 @@
+  form of <literal>myapp.example.com</literal>.
+ </para>
+ <para>
+  This chapter describes the changes required to allow applications to use a
+  separate domain.
+ </para>
+ <sect1 xml:id="sec.cap.customizing-app-domains">
+  <title>Customizing Application Domains</title>
+
+  <para>
+   Begin by adding the following to your
+   <filename>scf-config-values.yaml</filename>. Replace
+   <literal>APPDOMAIN</literal> with the domain to use with your applications:
+  </para>
+
+<screen>bosh:
+  instance_groups:
+  - name: api-group
+    jobs:
+    - name: cloud_controller_ng
+      properties:
+        app_domains:
+	- <replaceable>&lt;APPDOMAIN></replaceable>
+</screen>
+
+  <para>
+   If UAA is deployed, pass your UAA secret and certificate to SCF. Otherwise
+   deploy UAA first (See <xref linkend="sec.cap.install-uaa-prod"/>), then
+   proceed with this step:
+  </para>
+
+<screen>&prompt.user;SECRET=$(kubectl get pods --namespace uaa \
+-o jsonpath='{.items[?(.metadata.name=="uaa-0")].spec.containers[?(.name=="uaa")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}')
+
+&prompt.user;CA_CERT="$(kubectl get secret $SECRET --namespace uaa \
+-o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+</screen>
+
+  <para>
+   If this is an initial deployment, use <command>helm install</command> to
+   deploy SCF:
+  </para>
+
+<screen>&prompt.user;helm install suse/cf \
+--name susecf-scf \
+--namespace scf \
+--values scf-config-values.yaml \
+--set "secrets.UAA_CA_CERT=${CA_CERT}"
+</screen>
+
+  <para>
+   If this is an existing deployment, use <command>helm upgrade</command> to
+   apply the change:
+  </para>
+
+<screen>&prompt.user;helm upgrade susecf-scf suse/cf \
+--values scf-config-values.yaml --set "secrets.UAA_CA_CERT=${CA_CERT}"
+</screen>
+
+  <para>
+   Monitor the progess of the deployment:
+  </para>
+
+<screen>&prompt.user;watch -c 'kubectl get pods --namespace scf'</screen>
+
+  <para>
+   When all pods are in a ready state, do the following to confirm custom
+   application domains have been configured correctly.
+  </para>
+
+  <para>
+   Run <command>cf curl /v2/info</command> and verify the SCF domain is not
+   <literal>APPDOMAIN</literal>:
+  </para>
+
+<screen>&prompt.user;cf api --skip-ssl-validation https://api.example.com
+&prompt.user;cf curl /v2/info | grep endpoint
+</screen>
+
+  <para>
+   Deploy an application and examine the <literal>routes</literal> field to
+   verify <literal>APPDOMAIN</literal> is being used:
+  </para>
+
+<screen>&prompt.user;cf login
+&prompt.user;cf create-org org
+&prompt.user;cf create-space space
+&prompt.user;cf target -o org -s space
+&prompt.user;cf push myapp
+cf push myapp
+Pushing app myapp to org org / space space as admin...
+Getting app info...
+Creating app with these attributes...
+  name:       myapp
+  path:       /path/to/myapp
+  routes:
++   myapp.APPDOMAIN
+
+Creating app myapp...
```

Comment:
need more replaceable tags :) Every user-defined variable is lowercase & replaceable tags.